### PR TITLE
Infrastructure mapping (Amazon, Rancher)

### LINKED ISSUE #87: Map rancher services to Zoo services
We need a reliable way for relating Rancher services to Zoo services, we'll gather some metadata from there, like the users that have access to the service (from rancher members), which instances is it running on (we are using EC2s, so check the labels we have there) and which image is it using.

The most reliable link will probably be the docker image itself, which it has a direct relation with a GitLab repository.

### PR DESCRIPTION
**Pull request by paveldedik**
Closes #87, #89, #90

What is not included in this PR:

- GCP
- Cloudflare

### CODE REVIEW

File: zoo/base/settings.py [hunk 1]
```diff
@@ -36,6 +37,13 @@
     ZOO_AUDITING_DROP_ISSUES=(int, 7),
     ZOO_SONARQUBE_URL=(str, None),
     ZOO_SONARQUBE_TOKEN=(str, None),
+    AWS_CONFIG=(str, None),
+    AWS_CONFIG_FILE=(str, "/tmp/aws/config"),
+    AWS_SHARED_CREDENTIALS=(str, None),
+    AWS_SHARED_CREDENTIALS_FILE=(str, "/tmp/aws/credentials"),
+    RANCHER_API_URL=(str, None),
+    RANCHER_USERNAME=(str, None),
```
Comment (aexvir) [inline zoo/base/settings.py]: Username and password? Shouldn't it be `RANCHER_ACCESS_KEY` and `RANCHER_SECRET_KEY`?

Comment (paveldedik) [reply zoo/base/settings.py]: Yes, it should. I'll fix it.

File: zoo/datacenters/amazon.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    if names:
+        page_iterator = paginator.paginate(LoadBalancerNames=names)
+    else:
+        page_iterator = paginator.paginate()
+
+    for page in page_iterator:
+        yield from page["LoadBalancerDescriptions"]
+
+
+def iter_load_balancers_v2(names=None, profile_name="default"):
```
Comment (aexvir) [inline zoo/datacenters/amazon.py]: Couldn't this two be merged into one and passing the version as argument? Or it doesn't make sense?

Comment (paveldedik) [reply zoo/datacenters/amazon.py]: I think it makes sense. I will merge it.

File: zoo/services/models.py [hunk 3]
```diff
@@ -204,23 +200,3 @@ class Meta:
...
-        unique_together = ("provider", "region")
-
-    provider = models.CharField(max_length=100)
-    region = models.CharField(max_length=100, null=True, blank=True)
-
-    def __str__(self):
-        return f"{self.provider} {self.region}" if self.region else self.provider
-
-
-class ServiceQLSchema(DjangoQLSchema):
```
Comment (aexvir) [inline zoo/services/models.py:220]: This shouldn't have been deleted, just remove the datacenter related stuff

**Review by aexvir — APPROVED** [review]

**Pull request merged**